src/entities/activity/lib/services/AnswersUploadService.ts

```diff
@@ -118,14 +118,18 @@ class AnswersUploadService implements IAnswersUploadService {
   ): Promise<string> {
     const localFileExists = await FileSystem.exists(mediaFile.uri);
 
-    const logFileInfo = `(${mediaFile.type}, from answer #${logAnswerIndex})`;
+    let logFileInfo = `(${mediaFile.type}, from answer #${logAnswerIndex})`;
 
     if (!localFileExists) {
       throw new Error(
         `[UploadAnswersService.processFileUpload]: Local file ${logFileInfo} does not exist`,
       );
     }
 
+    const fileStat = await FileSystem.stat(mediaFile.uri);
+
+    logFileInfo += `, with size = ${fileStat.size} bytes`;
```

Comment:
Could we move the declaration of the `logFileInfo` variable here so that we pass the value once?